src/backend/Dockerfile.pangolin

```diff
@@ -41,10 +41,10 @@ RUN apt-get update && apt-get install -y  \
 # Install Pangolin and PangoLEARN + deps (for ncov)
 # The cov-lineages projects aren't available on PyPI, so install via git URLs.
 RUN pip3 install snakemake==6.8.0
-RUN pip3 install git+https://github.com/cov-lineages/pangolin.git@v4.1.3
-RUN pip3 install git+https://github.com/cov-lineages/pangolin-data.git@v1.16
-RUN pip3 install git+https://github.com/cov-lineages/scorpio.git@v0.3.17
-RUN pip3 install git+https://github.com/cov-lineages/constellations.git@v0.1.10
+RUN pip3 install git+https://github.com/cov-lineages/pangolin.git@v4.2
```

Comment:
this is the tool, the 3 repos below are where the lineage data is actually stored.

Reply:
Pangolin removed requirement on Snakemake version https://github.com/cov-lineages/pangolin/commit/3ecf8e9d845a0cecc1753054db42423aeba1555b. So if we want to remove this pin on pangolin.git, we may consider remove snakemake pin together.